src/api/system/system.ts

```diff
@@ -508,7 +508,7 @@ export default class System extends Base {
             downloadId: downloadId
         };
 
-        return this.wire.sendAction('download-runtime', data).then(() => undefined);
+        return this.wire.sendAction<any, any>('download-runtime', data).then(() => undefined);
```

Comment:
should <any, any> become <void, RuntimeDownloadOptions>?

Reply:
void, no because it might return something, we are typing the sendAction not the final value. RuntimeDownloadOptions could work, but I just typed it the same as the data object above, not sure if downloadId is part of RuntimeDownloadOptions

Reply:
Check my PR for updated signatures.